src/main/kotlin/org/rust/cargo/project/configurable/RustProjectConfigurable.kt

```diff
@@ -97,6 +113,8 @@ class RustProjectConfigurable(
         useCargoCheckAnnotator = settings.useCargoCheckAnnotator
         letBindingHint = HintType.LET_BINDING_HINT.enabled
         parameterHint = HintType.PARAMETER_HINT.enabled
+        lambdaParameterHint = HintType.LAMBDA_PARAMETER_HINT.enabled
+        smartHint = HintType.SMART_HINTING.get()
```

Comment:
I want to support settings in Clion for inlay parameter hints. But also support settings for inlay parameter like java or kotlin. You need to support getSupportedOptions.